lib/terraforming/resource/elastic_beanstalk_environment.rb

```diff
@@ -0,0 +1,87 @@
+module Terraforming
+  module Resource
+    class ElasticBeanstalkEnvironment
+      include Terraforming::Util
+
+      def self.tf(client: Aws::ElasticBeanstalk::Client.new)
+        self.new(client).tf
+      end
+
+      def self.tfstate(client: Aws::ElasticBeanstalk::Client.new)
+        self.new(client).tfstate
+      end
+
+      def initialize(client)
+        @client = client
+      end
+
+      def tf
+        apply_template(@client, "tf/ebenv")
+      end
+
+      def tfstate
+        environments.inject({}) do |resources, environment|
+          settings = settings_of(environment)
+          attributes = {
+            "name" => environment.environment_name,
+            "description" => environment.description ? environment.description : "",
+            "application" => environment.application_name,
+            "cname_prefix" => environment.cname.split(/\./).first,
+            "tier" => environment.tier.name,
+            "solution_stack_name" => environment.solution_stack_name,
+            # {}"template_name" => environment.template_name,
+            "settings.#" => settings.length.to_s
+            # {}"tag.#" => environment.tags.length.to_s
+            # no tags available with this call or any other at time of writing
+          }
+
+          settings.each do |setting|
+            #if setting.value != ""
```

Comment:
__[rubocop]__
* Incorrect indentation detected (column 12 instead of 14).
* Missing space after #. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-space)
* Incorrect indentation detected (column 14 instead of 12).

<!-- Comment by [SideCI](https://sideci.com) -->